docs/api.rst

```diff
@@ -11,39 +11,70 @@ Search
 ------
 
 .. autoclass:: Search
+   :inherited-members:
    :members:
 
 .. autoclass:: MultiSearch
+   :inherited-members:
+   :members:
+
+.. autoclass:: AsyncSearch
+   :inherited-members:
+   :members:
+
+.. autoclass:: AsyncMultiSearch
+   :inherited-members:
    :members:
```

Comment:
I believe the async API should be distinct from the sync API. I quite like the elasticsearch-py away, which is just to include the API at the bottom the page. But it could also be a different page.